app/src/openxr/cpp/DeviceDelegateOpenXR.cpp

```diff
@@ -1253,7 +1270,6 @@ DeviceDelegateOpenXR::DeleteLayer(const VRLayerPtr& aLayer) {
   }
   for (int i = 0; i < m.uiLayers.size(); ++i) {
     if (m.uiLayers[i]->GetLayer() == aLayer) {
-      m.uiLayers[i]->Destroy();
```

Comment:
I guess because it's redundant?

Reply:
yes; that's why I've added the Destroy call un the delete operator of the OpenXRLayerBase class